onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -1028,21 +1028,4 @@ def to_model_proto(
                 common_ops.common_opset.domain, common_ops.common_opset.version
             )
         )
-
-        try:
```

Comment:
maybe add a ` def to_model_proto(..., infer_shape=False)` in which this step is skipped by default instead of deleting it?

Reply:
Maybe we should assume the converter produces correct onnx graphs only. It fails if it cannot. In that case, more checkings seem unnecessary and this code should be removed or called only in the unit tests.

Reply:
I added the shape inference step in test instead.